src/dso_api/dynamic_api/middleware.py

```diff
@@ -98,6 +98,16 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             method=request.method,
             request_headers=repr(request.META),
             subject=subject,
-            data=data)
+            data=data,
+        )
 
         audit_log.info(json.dumps(log))
+
+
+class QueryParamLegalityCheckMiddleware(BaseMiddleware):
+    def process_view(self, request, *args, **kwargs):
```

Comment:
Heeft het volgens jou nog een meerwaarde om een toelichting toe te voegen aan deze class? 
"""...
"""

Reply:
ja zeker, zit er inmiddels in